taichi/ir/ir.cpp

```diff
@@ -819,6 +819,7 @@ std::unique_ptr<Block> Block::clone() const {
 DelayedIRModifier::~DelayedIRModifier() {
   TI_ASSERT(to_insert_before.empty());
   TI_ASSERT(to_erase.empty());
+  TI_ASSERT(to_replace_with.empty());
```

Comment:
Also `to_insert_after` here :-) I forgot to add it when I added the member variable `to_insert_after`.